walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,48 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+        //throw new UnsupportedOperationException("TODO: Implement this method.");
+        List<TimeRange> blockedTimes = new ArrayList<TimeRange>();
```

Comment:
You could consider not declaring blockedTimes until after line 34, since you might return from the function before you need to use it. In this case it doesn't matter too much since an empty List won't take up much space, but for future reference, this could be a big memory saver if this was a larger object.